test/alternator/test_streams.py

```diff
@@ -677,8 +677,6 @@ def do_updates(table, p, c):
 
 # Test OLD_IMAGE using UpdateItem. Verify that the OLD_IMAGE indeed includes,
 # as needed, the entire old item and not just the modified columns.
-# Currently fails in Alternator because the item's key is missing in OldImage (#6935).
```

Comment:
Good you removed this comment, but please replace it by a comment like "Reproduces issue #6935".

Reply:
sure

Reply:
Fixed